projects/ccf-rui/src/app/shared/components/block-size-input/block-size-input.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+
+export interface TissueData {
+  width: number;
+  height: number;
+  depth: number;
+}
+
+/**
+ * Component for entering dimensions of the tissue block
+ */
+@Component({
+  selector: 'ccf-block-size-input',
+  templateUrl: './block-size-input.component.html',
+  styleUrls: ['./block-size-input.component.scss']
+})
+export class BlockSizeInputComponent {
+
+  /**
+   * Values of block dimensions to be emitted
+   */
+  tissueData: TissueData = {
+    width: 10,
```

Comment:
Should be x/y/z. That data is/will be stored in x/y/z format, so best to just keep it that way everywhere for ease of movement.